keras_nlp/models/bert/bert_masked_lm_test.py

```diff
@@ -104,13 +104,15 @@ def test_serialization(self):
     @pytest.mark.large  # Saving is slow, so mark these large.
     def test_saved_model(self, save_format, filename):
         model_output = self.masked_lm.predict(self.raw_batch)
-        save_path = os.path.join(self.get_temp_dir(), filename)
-        self.masked_lm.save(save_path, save_format=save_format)
-        restored_model = keras.models.load_model(save_path)
+        path = os.path.join(self.get_temp_dir(), filename)
+        # Don't save traces in the tf format, we check compilation elsewhere.
+        kwargs = {"save_traces": False} if save_format == "tf" else {}
+        self.masked_lm.save(path, save_format=save_format, **kwargs)
+        restored_model = keras.models.load_model(path)
 
         # Check we got the real object back.
         self.assertIsInstance(restored_model, BertMaskedLM)
 
         # Check that output matches.
         restored_output = restored_model.predict(self.raw_batch)
-        self.assertAllClose(model_output, restored_output)
+        self.assertAllClose(model_output, restored_output, atol=0.01, rtol=0.01)
```

Comment:
Curious - why would it fluctuate? `0.01` without context is not a common scale in `atol`.

Reply:
I'm not really sure, also this is totally unrelated, so I can split into a separate PR. I was seeing enough fluctuation on my nvidia GPU that sometimes these tests would fail. This is true on master too.

It looks like it is just a precision issue, the floats are the same, with a lax enough tolerance.